examples/basic-otlp fails to compile

<details>
<summary>Click to expand</summary>
<pre>
error[E0277]: the trait bound `opentelemetry::exporter::metrics::stdout::ExportBatch: serde::ser::Serialize` is not satisfied
   --> examples\basic-otlp\src\main.rs:29:34
    |
29  |             serde_json::to_value(batch)
    |                                  ^^^^^ the trait `serde::ser::Serialize` is not implemented for `opentelemetry::exporter::metrics::stdout::ExportBatch`
    | 
   ::: C:\Users\gogam\.cargo\registry\src\github.com-1ecc6299db9ec823\serde_json-1.0.59\src\value\mod.rs:962:8
    |
962 |     T: Serialize,
    |        --------- required by this bound in `serde_json::to_value`

error: aborting due to previous error
</pre>
</details>

I believe the simplest way to fix this is to add `serialize` feature to Cargo.toml

```
opentelemetry-otlp = { path = "../../opentelemetry-otlp", features = ["serialize"] }
```

--- a/examples/basic-otlp/Cargo.toml
+++ b/examples/basic-otlp/Cargo.toml
@@ -6,7 +6,7 @@ edition = "2018"
 [dependencies]
 futures = "0.3"
 lazy_static = "1.4"
-opentelemetry = { path = "../../", features = ["tokio", "metrics"] }
+opentelemetry = { path = "../../", features = ["tokio", "metrics", "serialize"] }
 opentelemetry-otlp = { path = "../../opentelemetry-otlp" }
 serde_json = "1.0"
 tokio = { version = "0.2", features = ["full"] }

